### PR TITLE
Adding call to send account deletion confirmation email

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/h8nV6an7/883-user-account-creation-account-deletion-request

### CODE REVIEW

File: ras_party/controllers/respondent_controller.py [hunk 1]
```diff
@@ -123,6 +125,31 @@ def delete_respondents_marked_for_deletion(session):
         session.query(BusinessRespondent).filter(BusinessRespondent.respondent_id == respondent.id).delete()
         session.query(PendingEnrolment).filter(PendingEnrolment.respondent_id == respondent.id).delete()
         session.query(Respondent).filter(Respondent.id == respondent.id).delete()
+        send_account_deletion_confirmation_email(respondent.email_address, respondent.first_name)
+
+
+def send_account_deletion_confirmation_email(email_address, name):
```
Comment (insacuri) [inline ras_party/controllers/respondent_controller.py]: Put type hints here
```python
def send_account_deletion_confirmation_email(email_address: str, name: str):
```

File: ras_party/controllers/respondent_controller.py [hunk 2]
```diff
@@ -123,6 +125,31 @@ def delete_respondents_marked_for_deletion(session):
...
+    """
+    Sends email notification for account deletion confirmation.
+    :param email_address:
+    :type email_address:
```
Comment (insacuri) [inline ras_party/controllers/respondent_controller.py]: Can you tidy this bit up?  The 2 types aren't needed (as they're typehints in the function) and rtype can go too as this function doesn't return anything

**Review by insacuri — CHANGES_REQUESTED** [review]
2 small changes, but looks good

File: ras_party/controllers/respondent_controller.py (hunk 2 above)
Comment (creativeappdevamit) [reply ras_party/controllers/respondent_controller.py]: done

File: ras_party/controllers/respondent_controller.py (hunk 1 above)
Comment (creativeappdevamit) [reply ras_party/controllers/respondent_controller.py]: done

**Review by insacuri — APPROVED** [review]

**Pull request merged**